### ISSUE TITLE
Polymer UI issues in 2.10

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#5241]
In 2.10 testing I noticed that when navigating to the UI for the first time (via nginx, in the odpi-egeria-lab help deployment) I very briefly get a graphical 'forbidden' page, before being redirected to login.

This is minor, but slightly offputting. It's not to easily capture a screenshot as very brief

Is this intentional?
Should something be added to clarify in the docs?

I also notice a strange UI layout after login (chrome 91.0.4472.77)
<img width="1363" alt="Screenshot 2021-05-28 at 10 27 44" src="https://user-images.githubusercontent.com/7292002/119963574-26b4fc80-bfa0-11eb-951a-07b2e9183701.png">

After a search for 'csv'  I get:

<img width="1275" alt="Screenshot 2021-05-28 at 10 23 06" src="https://user-images.githubusercontent.com/7292002/119963710-5106ba00-bfa0-11eb-9cb9-6dd3f80e891d.png">

The static UI content being used is 2.8.0 as per https://github.com/odpi/egeria/issues/5211

A search for 'Week' (after running the Building a data catalog notebook) returned reasonable results.

Only asset catalog is available so unable to test any other aspects.

**Comment by planetf1** [comment]
I have modified the helm charts to inject a file 'application.properties' into the container.

However it is unclear where this file should go. I have placed into /var/www but this does not seem to have any effect
ie:

```
$ kubectl exec -it lab-odpi-egeria-lab-uistatic-76d57d5888-75rg6 -- /bin/sh       [16:57:02]
# cd /var/www
# ls -la
total 56
drwxr-xr-x. 1 root root 4096 May 28 15:53 .
drwxr-xr-x. 1 root root 4096 May 27 08:30 ..
-rw-r--r--. 1 root root  169 May 27 08:29 about.json
-rw-r--r--. 1 root root  481 May 28 15:53 application.properties
drwxr-xr-x. 3 root root 4096 May 27 08:30 images
-rw-r--r--. 1 root root 1692 May 27 08:29 index.html
drwxr-xr-x. 2 root root 4096 May 27 08:30 locales
-rw-r--r--. 1 root root  466 May 27 08:25 manifest.json
drwxr-xr-x. 2 root root 4096 May 27 08:30 new
drwxr-xr-x. 6 root root 4096 May 27 08:30 node_modules
-rw-r--r--. 1 root root  188 May 27 08:25 service-worker.js
drwxr-xr-x. 3 root root 4096 May 27 08:30 themes
# cat application.properties
# SPDX-License-Identifier: Apache-2.0
# Copyright Contributors to the Egeria project.
title = 'Open Metadata Platform:|Find the right data with governance. (dev)'
description = 'Have a question?||Visit https://github.com/odpi/egeria .@@What is Open Metadata?||Start your customer journey with this link to know what is Open Metadata and how can you use this applicaiton.@@Have more cool ideas?||Feel free to let us know your ideas and business challenges so we can make it better.'#
```
with the result being the same UI issue

I then reviewed the source at https://github.com/odpi/egeria-ui/blob/74696b552562c2ccc99127fe1ac6d6b5cd7500b1/new/egeria-home.component.js & it looks as if the current code is in the 'new' subdir.
I moved the file into /var/www/new/application.properties but this did not appear to work either
Also added a new line - no difference

(not looked at docker compose yet)

**Comment by planetf1** [comment]
See https://github.com/odpi/egeria/pull/5236 for PR which both sets the versions as needed for the release, and then adds application.properties. 
Docker config is still needed, but I need confirmation as to where this 'application.properties' should really go @sarbull @lpalashevski @bogdan-sava

**Comment by sarbull** [comment]
> See #5236 for PR which both sets the versions as needed for the release, and then adds application.properties.
> Docker config is still needed, but I need confirmation as to where this 'application.properties' should really go @sarbull @lpalashevski @bogdan-sava

it should go in the ui-chassis's application.properties as @lpalashevski mentioned above..

**Comment by lpalashevski** [comment]
> See #5236 for PR which both sets the versions as needed for the release, and then adds application.properties.
> Docker config is still needed, but I need confirmation as to where this 'application.properties' should really go @sarbull @lpalashevski @bogdan-sava 

Applogies. This is incomplete implementation from our side, we should this on master today, test it and let you know so you can cherry pick the fix in #5236. No need to configure ui-chassis-spring spearately for the chars/compose deployments but get these as default configuration like it was until now for the other app properties.

**Comment by planetf1** [comment]
Thanks. I've tested release 2.10 in k8s with the fix and it looks good now.

I'll leave this to close as you wish (depending on if you need any further changes).
